src/encoded/tests/test_renderers.py

```diff
@@ -20,8 +25,53 @@ def __init__(self, content_type=None, status_code: int = 200, json=None, content
         super().__init__(status_code=status_code, json=json, content=content, url=url)
 
 
+def test_mime_variables():
+
+    # Really these don't need testing but it's useful visually to remind us of their values here.
+    assert MIME_TYPE_HTML == 'text/html'
+    assert MIME_TYPE_JSON == 'application/json'
+    assert MIME_TYPE_LD_JSON == 'application/ld+json'
+
+    # The MIME_TYPES_SUPPORTED is a list whose first element has elevated importance as we've structured things.
+    # First check that it is a list, and that its contents contain the things we support. That isn't controversial.
+    assert isinstance(MIME_TYPES_SUPPORTED, list)
+    assert set(MIME_TYPES_SUPPORTED) == {MIME_TYPE_JSON, MIME_TYPE_HTML, MIME_TYPE_LD_JSON}
+    # Check that the first element is consistent with the MIME_TYPE_DEFAULT.
+    # It's an accident of history that this next relationship matters, but at this point check for consistency.
+    assert MIME_TYPE_DEFAULT == MIME_TYPES_SUPPORTED[0]
+    # Now we concern ourselves with the actual values...
+    # TODO: I think it's a bug that JSON is at the head of this list (and so the default) in cgap-portal.
+    #       cgap-portal needs to be made to match what Fourfront does to dig it out of a bug I introduced.
+    #       -kmp 29-Jan-2022
+    assert MIME_TYPES_SUPPORTED == [MIME_TYPE_HTML, MIME_TYPE_JSON, MIME_TYPE_LD_JSON]
+    assert MIME_TYPE_DEFAULT == MIME_TYPE_HTML
```

Comment:
These two lines in this test are different than in `cgap-portal` because they have to correspond to differences I had to make in `renderers.py` to make this a non-breaking change.